charts/incubator/amd-automatic-music-downloader/Chart.yaml

```diff
@@ -0,0 +1,32 @@
+apiVersion: v2
+appVersion: "latest"
+dependencies:
+- name: common
+  repository: https://truecharts.org
+  version: 8.15.2
+deprecated: false
+description: A Lidarr companion script to automatically download music for Lidarr.
+home: https://github.com/truecharts/apps/tree/master/charts/incubator/amd-automatic-music-downloader
+icon: https://truecharts.org/_static/img/appicons/amd-automatic-music-downloader-icon.png
+keywords:
+- amd
```

Comment:
Using world-size-brands as keywords for an App that is super niche, is not really best practice
```suggestion
```